src/main/java/dev/amble/ait/core/tardis/manager/old/DeprecatedServerTardisManager.java

```
@@ -152,9 +152,9 @@ public void forEach(Consumer<ServerTardis> consumer) {
         this.lookup.forEach((uuid, either) -> either.ifLeft(consumer));
     }
 
-    @Nullable private Either<ServerTardis, Exception> loadTardis(MinecraftServer server, UUID uuid) {
+    private Either<ServerTardis, Exception> loadTardis(MinecraftServer server, UUID uuid) {
         if (this.fileManager.isLocked())
-            return null;
+            return Either.right(new Exception("TardisFileManager is locked!"));
```

uhhhhhhhhhhhhhhhhhhhhhhh
no.
thats not how it should be done

this method should return null
but it should be handled properly wherever it gets called
its not private for no reason

---

if it puts an exception inside the map it wont ever try to load that tardis
so if, for example, the TM is locked and then some BlockEntity tries to load its tardis instance it will silently fail which is expected
but when it will try again, it will check the map, find an exception and go on with its day thinking that there was no tardis bound to begin with